http/src/main/resources/reference.conf

```diff
@@ -478,6 +478,12 @@ gke {
       {
         name = "cromwell-service"
         kind = "ClusterIP"
+        path = "/swagger/index.html?url=/proxy/{proxyPath}/swagger/cromwell.yaml"
+      },
+      {
+        name = "job-manager"
+        kind = "ClusterIP"
+        path = "/jobs"
```

Comment:
This currently doesn't work because CromwellApp doesn't yet have a reachable JobManager. That work is in https://broadworkbench.atlassian.net/browse/BT-619 which I am starting work on now. Assume that BT-619 will merge before this.

Reply:
how will the 2 URLs returned to user? looks like `getProxyUrls` is still returning one URL

Also in APIs, we're only returning one proxy URL for a given app

Reply:
`getProxyUrls` returns `Map[ServiceName, URL]`. Since each `App` can have multiple services and each service has its own URL, apps can have multiple proxy URLs.

The UI is using the `apps` API, specifically https://leonardo.dsde-dev.broadinstitute.org/#/apps/listAppByProject which returns a `proxyUrls` object that contains the service name to URL mapping returned by `getProxyUrls`.

Reply:
It sounds like this is the first time the `proxyUrls` map will contain more than one item. Should we be concerned about unexpected behavior from the UI, which will need to be prepared to select one of several URLs? I can imagine something on the frontend checking that the map contains a single item and throwing an error if not.

Reply:
There are 3 non-test reads of `proxyUrls` and they all ask for specific keys (`galaxy` or `cromwell-service`) so adding an additional entry should not break anything.